src/pages/Register/Initial/index.jsx

```diff
@@ -0,0 +1,278 @@
+import NumberFormat from "react-number-format";
+
+import styles from "./styles.module.css";
+
+export function RegisterInitial() {
+  const history = useHistory();
+
+  const { newCommerce, setNewCommerce } = useRegister();
+  const { nextStep } = useStep();
+
+  const validationSchema = yup.object().shape({
+    name: yup.string().required(requiredInfoAlert),
+    email: yup.string().email("Email inválido.").required(requiredInfoAlert),
+    phone: yup
+      .string()
+      .required(requiredInfoAlert)
+      .min(11, "Telefone inválido."),
+    cnpj: yup.string().required(requiredInfoAlert).min(16, "CNPJ inválido."),
+    instagram: yup.string(),
+    facebook: yup.string(),
+    city: yup.string().required(requiredInfoAlert),
+    district: yup.string(),
+    street: yup.string().required(requiredInfoAlert),
+    number: yup.number().required(requiredInfoAlert),
+    password: yup
+      .string()
+      .min(6, "Senha muito curta.")
+      .max(18, "Senha muito longa.")
+      .required(requiredInfoAlert),
+    passwordConfirm: yup
+      .string()
+      .oneOf([yup.ref("password"), null], "As senhas devem ser iguais."),
+  });
+
+  const { values, errors, touched, handleChange, handleBlur, handleSubmit } =
+    useFormik({
+      onSubmit: async (values) => {
+        setNewCommerce({
+          ...newCommerce,
+          name: values.name,
+          phone: values.phone,
+          cnpj: values.cnpj,
+          password: values.password,
+          social_medias: {
+            email: values.email,
+            instagram: values.instagram,
+            facebook: values.facebook,
+          },
+          address: {
+            city: values.city,
+            street: values.street,
+            district: values.district,
+            number: values.number,
+          },
+        });
+        nextStep();
+      },
+
+      validationSchema: () => validationSchema,
+      initialValues: {
+        name: newCommerce.name,
+        phone: newCommerce.phone,
+        email: newCommerce.social_medias.email,
+        cnpj: newCommerce.cnpj,
+        instagram: newCommerce.social_medias.instagram,
+        facebook: newCommerce.social_medias.facebook,
+        city: newCommerce.address.city,
+        district: newCommerce.address.district,
+        street: newCommerce.address.street,
+        number: newCommerce.address.number,
+        password: "",
+        passwordConfirm: "",
+      },
+    });
+
+  return (
+    <div className={styles.container}>
+      <section className={styles.header}>
+        <h1>Seja bem-vinde ao SoLanches!</h1>
+        <div className={styles.description}>
+          <span>Atenção:</span>
+          <span>
+            O cadastro só é necessário para usuários que querem vender algo.
+          </span>
+        </div>
+      </section>
+      <section className={styles.inputGroup}>
+        <form onSubmit={() => handleSubmit()}>
+          <div className={styles.name}>
+            <label htmlFor="name">Nome</label>
```

Comment:
Adicionar o * vermelho onde o campo for obrigatório

Reply:
Criar variável para o vermelho, pois ele será utilizado aqui e no card que comentei mais abaixo.